src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -335,6 +340,16 @@ private void processProducer(RoutingContext routingContext) {
         }
     }
 
+    private void healthy(RoutingContext routingContext) {
+        HttpResponseStatus httpResponseStatus = this.healthChecker.isHealthy() ? HttpResponseStatus.OK : HttpResponseStatus.INTERNAL_SERVER_ERROR;
```

Comment:
Is 500 really the best choice? 500 does sort of make sense for healthy, but for readiness 500 is a little ambiguous and 404 would be a better fit, no?

Reply:
I put 500 but I was not so happy about that. The 404 is returned when I resource doesn't exist, so it could fit well but it is also returned when a GET is done on a not existing endpoint. For example, for the HTTP bridge if you do a GET on the root "/" it returns 404 because there is no Vert.x route handling that (no endpoint exposed on the OpenAPI spec). So could it make confusion?

Reply:
Yeah, I wondered about that, but I don't think it really makes a difference. I don't think there's any diagnostic value in having them be different.